test/bacom-blog/quote/quote.test.js

```diff
@@ -0,0 +1,72 @@
+import { convertPullQuote, QUOTE_BLOCK_NAME } from '../../../bacom-blog/pull-quote-update.js';
+import { readFile } from 'fs/promises';
+import { expect } from '@esm-bundle/chai';
+import { getMdast } from '../../../utils/mdast-utils.js';
+
+describe('convertPullQuote', () => {
+  it('adds the correct heading', async () => {
+    const pullQuotePath = new URL('./mocks/single-quote.md', import.meta.url);
+    const pullQuoteMd = await readFile(pullQuotePath, 'utf-8');
+    const pullQuoteMdast = await getMdast(pullQuoteMd);
+
+    await convertPullQuote(pullQuoteMdast);
+
+    const mdastBody  = pullQuoteMdast.children[0].children[0];
+    const headerRow = mdastBody.children[0];
+    const textNode = headerRow.children[0].children[0].children[0];
```

Comment:
nit: I would prefer select for readability.
```suggestion
    const mdastBody  = select("gtBody", pullQuoteMdast);
    const headerRow = mdastBody.children[0];
    const textNode = select("gtCell paragraph text", headerRow);
```

Reply:
changed elsewhere where applicable!